salt/cloud/clouds/vmware.py

```diff
@@ -27,23 +27,25 @@
 .. code-block:: yaml
 
     my-vmware-config:
-      driver: vmware
-      user: "DOMAIN\\user"
-      password: "verybadpass"
-      url: "vcenter01.domain.com"
-
-    vmware-vcenter02:
-      driver: vmware
-      user: "DOMAIN\\user"
-      password: "verybadpass"
-      url: "vcenter02.domain.com"
-
-    vmware-vcenter03:
-      driver: vmware
-      user: "DOMAIN\\user"
-      password: "verybadpass"
-      url: "vcenter03.domain.com"
-      protocol: "http"
+      provider: vmware
```

Comment:
@nmadhok - these references need to be `driver: vmware` in all of these examples, instead of `provider`.